src/makeCalculator.js

```diff
@@ -37,7 +37,37 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    operate(opetation, number) {
+      opetation(number, this);
+
+      return this;
+    },
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    add(number, that) {
+      that.result += number;
+    },
+
+    subtract(number, that) {
+      that.result -= number;
+    },
+
+    multiply(number, that) {
+      that.result *= number;
+    },
+
+    divide(number, that) {
+      that.result /= number;
+    },
```

Comment:
Functions should only take one parameter and add it to the current result.

Reply:
I had made it to bind the context, but the call method fix this issue